### ISSUE TITLE
intltool build breaks following perl upgrade

### ISSUE THREAD

**Issue by waddlesplash** [haikuports/haikuports#1518]
```
checking for XML::Parser... configure: error: XML::Parser perl module is required for intltool
```
cc @korli

**Comment by waddlesplash** [comment]
full log at http://vmpkg.haiku-os.org/kitchen/#/build/29

**Comment by pulkomandy** [comment]
We have a recipe for it:
https://github.com/haikuports/haikuports/tree/master/dev-perl/xml_parser

Perl packages store files in perl-version specific dirs. So everything in dev-perl needs to be rebuilt after an update of perl. Bump revisions there and retry?

**Comment by korli** [comment]
What isn't working is, a xml_parser hpkg built for perl 5.18.2 shouldn't be selected by haikuporter when using perl 5.26. Seems the compatibility string is missing, will have a look.

**Comment by korli** [comment]
I changed the vendor_perl compatibility in a854d846b0eb33b04e8ea5b34ad104d72bbd68a9
Please be aware that the perl should be rebuilt first (and for me installed too), then depending packages in need of a rebuild. Otherwise they could be built with an old perl version.

**Comment by korli** [comment]
intltool still needs to be patched for perl 5.26 at runtime ...

**Comment by korli** [comment]
Fixed in 44e5fa47f1e480ea49a84fe00aee7a95ae67b38b
